discordpp/botStruct.hh

```diff
@@ -38,12 +39,12 @@ class BotStruct {
 
   protected:
     virtual void runctd() {
-        std::cerr << "Starting run loop" << '\n';
+        log::log(log::error, [](std::ostream* log) { *log << "Starting run loop" << '\n'; });
         work = std::make_unique<boost::asio::executor_work_guard<
             boost::asio::io_context::executor_type>>(
             boost::asio::make_work_guard(*aioc));
         aioc->run();
-        std::cerr << "Ending run loop" << '\n';
+        log::log(log::error, [](std::ostream* log) { *log << "Ending run loop" << '\n'; });
```

Comment:
Hmm... These should either be left as-is. `'\n'` could be moved into the strings though.